homeassistant/components/folder_watcher/strings.json

```
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+    },
+    "error": {
+      "not_dir": "Configured path is not a directory",
+      "not_readable_dir": "Configured path is not readable",
+      "not_allowed_dir": "Configured path is not in allowlist"
+    },
+    "step": {
+      "user": {
+        "description": "Press submit for initial setup. On the created config entry, press configure to add sensors for selected processes",
+        "data": {
+          "folder": "Path to the watched folder",
+          "patterns": "Pattern(s) to monitor"
+        },
+        "data_description": {
+          "folder": "Path needs to be from root, as example `/config`",
+          "patterns": "Example is `*.yaml` to only see yaml files"
+        }
+      }
+    }
+  },
+  "issues": {
```

```suggestion
  },
    "options": {
    "step": {
        "init": {
            "data": {
                "patterns": "[%key:component::folder_watcher::config::step::user::data::patterns%]"
            },
            "data_description": {
                "patterns": "[%key:component::folder_watcher::config::step::user::data_description::patterns%]"
            }
        }
    }
  },
  "issues": {
```
The option flow needs the strings as well.